src/Leela.cpp

```diff
@@ -63,15 +63,15 @@ static void parse_commandline(int argc, char *argv[]) {
                        "Requires --noponder.")
         ("visits,v", po::value<int>(),
                      "Weaken engine by limiting the number of visits.")
-        ("lagbuffer,b", po::value<int>()->default_value(cfg_lagbuffer_cs),
+        ("lagbuffer,b", po::value<int>(),//->default_value(cfg_lagbuffer_cs),
```

Comment:
Why this change?

Reply:
The default is set in GTP::setup_default_parameters() called in main(); using `po::value<int>()->default_value(cfg_lagbuffer_cs)` seems to mean that if there isn't --lagbuffer in the command, cfg_lagbuffer_cs will be replaced by the default value even though its value is changed inside parse_commandline(), and I find this confusing. For example, I changed cfg_resignpct to 0 when --handicap is enabled, but the change didn't take effect until I removed `->default_value(cfg_resignpct)`.

Reply:
The reason to have the ->default_value there is that they are shown in the command line help, so you can't just remove them.